src/test/groovy/pl/cleankod/BaseApplicationSpecification.groovy

```diff
@@ -17,6 +20,10 @@ abstract class BaseApplicationSpecification extends Specification {
   @SuppressWarnings('unused')
   def setupSpec() {
     if (init) {
+      SimpleModule module = new SimpleModule()
+      module.addDeserializer(Account.class, new AccountDeserializeService())
+      objectMapper.registerModule(module)
```

Comment:
Isn't there a better way to deal with this? Otherwise we'd have to always have the `ObjectMapper`'s config doubled.